src/main/resources/sql/CreateAppDb.sql

```diff
@@ -96,6 +96,12 @@ CREATE TABLE IF NOT EXISTS resource_link (
         CHECK ((collection_fk is NULL) or (content_fk is NULL))
 );
 
+CREATE TABLE IF NOT EXISTS subtree_has_resource (
+  collection_fk       INTEGER NOT NULL REFERENCES collection_entity (id) ON DELETE CASCADE,
+  dublin_core_fk      INTEGER NOT NULL REFERENCES dublin_core_entity (id) ON DELETE CASCADE,
+  PRIMARY KEY (collection_fk, dublin_core_fk) ON CONFLICT IGNORE
+);
+
```

Comment:
This is a denormalized table that lets us quickly look up which helps resources are available for a collection or any of its children.

Reply:
I just did a quick search on denormalized tables. By this you mean that the intention of the table is to avoid digging deep into multiple tables to see if there are resources for a collection, right?

Reply:
Also why "ON CONFLICT IGNORE"?